tests/test_targeted.yml

```diff
@@ -39,23 +60,38 @@
     - preprocessing
   files:
     - path: results/csv/markduplicates.csv
+      md5sum: 0d6120bb99e92f6810343270711ca53e
     - path: results/csv/markduplicates_no_table.csv
+      md5sum: 2a2d3d4842befd4def39156463859ee3
     - path: results/csv/recalibrated.csv
+      md5sum: 42628ec994c16f565e5407b40a9c1ac3
     - path: results/multiqc
     - path: results/preprocessing/markduplicates/test/test.md.cram
```

Comment:
Also here and in some other files. Would be nice to be consistent with the comment.

Reply:
Now I tried adding those comments everywhere for consistency.